bl/validation/k8sValidator.go

```diff
@@ -38,16 +41,17 @@ func (val *K8sValidator) ValidateResources(filesConfigurationsChan chan *extract
 
 		for fileConfigurations := range filesConfigurationsChan {
 
-			isValid, validationErrors, err := val.validateResource(fileConfigurations.FileName)
+			isValid, validationErrors, err, validationWarning := val.validateResource(fileConfigurations.FileName)
```

Comment:
I think you should return the error object first or last, though I can't find any reference for that practice I think it makes the usage of errors more organized

Reply:
@dimabru btw, found it, https://gobyexample.com/errors by convention errors are the last return value